Gemfile

```
@@ -25,6 +25,11 @@
 gem 'view_component-contrib', '~> 0.2.3'
 gem 'simple_form'
 
+# Tracking
+gem "stackprof"
```

rubocop

[convention] Prefer single-quoted strings when you don't need string interpolation or special symbols. (https://rubystyle.guide#consistent-string-literals)

(Style/StringLiterals)